src/core/Akka.Remote/Transport/TestTransport.cs

```diff
@@ -64,27 +62,29 @@ public TestTransport(Address localAddress, AssociationRegistry registry, long ma
                 new SwitchableLoggedBehavior<bool, Tuple<Address, TaskCompletionSource<IAssociationEventListener>>>(
                     x => DefaultListen(), x => _registry.LogActivity(new ListenAttempt(LocalAddress)));
             AssociateBehavior =
-                new SwitchableLoggedBehavior<Address, AssociationHandle>(DefaultAssociate, address => registry.LogActivity(new AssociateAttempt(LocalAddress, address)));
-            ShutdownBehavior = new SwitchableLoggedBehavior<bool, bool>(x => DefaultShutdown(), x => registry.LogActivity(new ShutdownAttempt(LocalAddress)));
+                new SwitchableLoggedBehavior<Address, AssociationHandle>(DefaultAssociate,
+                    address => registry.LogActivity(new AssociateAttempt(LocalAddress, address)));
+            ShutdownBehavior = new SwitchableLoggedBehavior<bool, bool>(x => DefaultShutdown(),
+                x => registry.LogActivity(new ShutdownAttempt(LocalAddress)));
             DisassociateBehavior = new SwitchableLoggedBehavior<TestAssociationHandle, bool>(x =>
             {
                 DefaultDisassociate(x);
-                return Task.Run(() => true);
+                return Task.FromResult(true);
             }, remote => _registry.LogActivity(new DisassociateAttempt(remote.LocalAddress, remote.RemoteAddress)));
 
             WriteBehavior = new SwitchableLoggedBehavior<Tuple<TestAssociationHandle, ByteString>, bool>(
                 args => DefaultWriteBehavior(args.Item1, args.Item2),
-                data => _registry.LogActivity(new WriteAttempt(data.Item1.LocalAddress, data.Item1.RemoteAddress, data.Item2)));
+                data =>
+                    _registry.LogActivity(new WriteAttempt(data.Item1.LocalAddress, data.Item1.RemoteAddress, data.Item2)));
```

Comment:
I believe that this line may be behind some of the odd performance numbers I'm seeing in successive runs of the same spec - as the underlying `ConcurrentStack` used to log all of the writes to the transport grows I believe it begins to impact the throughput of the test itself. We should come up with a HOCON configuration switch or something to disable logging for `TestTransport` if there isn't one already.